README.md

```diff
@@ -1,85 +1,151 @@
-# Fury Kubernetes Logging
+<h1>
+    <img src="https://github.com/sighupio/fury-distribution/blob/master/docs/assets/fury-epta-white.png?raw=true" align="left" width="90" style="margin-right: 15px"/>
+    Kubernetes Fury Logging
+</h1>
 
-This repo contains logging components to deploy on top of Kubernetes. Fury's
-logging stack is based on Elasticsearch, a very popular open-source search
-engine commonly used for log analytics. Other components are used in integration
-with Elasticsearch.
+![Release](https://img.shields.io/github/v/release/sighupio/fury-kubernetes-logging?label=Latest%20Release)
+![License](https://img.shields.io/github/license/sighupio/fury-kubernetes-logging?label=License)
+![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack&label=Slack)
 
-## Requirements
+<!-- <KFD-DOCS> -->
 
-All packages in this repository have following dependencies, for package
-specific dependencies please visit the single package's documentation:
+**Kubernetes Fury Logging** provides a logging stack for the [Kubernetes Fury Distribution (KFD)][kfd-repo].
 
-- [Kubernetes](https://kubernetes.io) >= `v1.18.0`
-- [Furyctl](https://github.com/sighup-io/furyctl) package manager to install Fury packages
-- [Kustomize](https://github.com/kubernetes-sigs/kustomize) >= `v3`
+If you are new to KFD please refer to the [official documentation][kfd-docs] on how to get started with KFD.
 
+## Overview
 
-## Examples
+**Kubernetes Fury Logging** uses a collection of open source tools to provide the most resilient and robust logging stack for the cluster. 
```

Comment:
trailing white space here make the linter sad

Reply:
done